src/main/java/com/talentpath/shamazin/showItemPage/models/Item.java

```diff
@@ -105,16 +108,34 @@ public Item(@NotBlank ItemFamily itemFamily, List<ProductPhoto> productPhotos, @
         this.isPrimeEligible = isPrimeEligible;
     }
 
+    public Item(@NotBlank ItemFamily itemFamily, List<ProductPhoto> productPhotos, List<Property> properties, @NotBlank String name, @NotBlank Double price, @NotBlank Integer stockRemaining, @NotBlank Boolean isPrimeEligible) {
+        this.itemFamily = itemFamily;
+        this.productPhotos = productPhotos;
+        this.properties = properties;
```

Comment:
Why do we have two identical Item constructors (with only "this.properties = properties" being the distinction)? Can't we just move this into the constructor above, or am I missing something?

Reply:
I just did that because I didn't want to refactor my unit tests that used the other constructor, but I can change it if you think it is necessary